### DISCUSSION TITLE
Possible to change port on Docker install?

### DISCUSSION THREAD

**Discussion by mariusCZ** [firefly-iii/firefly-iii#6792] [Firefly III] [answered]
Hello,

When I try to run Firefly with Docker, I get the following error:

(1)Operation not permitted: AH00072: make_sock: could not bind to address [::]:8080
(1)Operation not permitted: AH00072: make_sock: could not bind to address 0.0.0.0:8080

I am trying to set this up on a shared server so I am rootless, so as I understand, this port is simply just reserved.  My question is, how can I change the port? I change the port within the docker compose file, but that is not enough. Here is my docker file:

```services:

  app:

    image: fireflyiii/core:latest

    restart: always

    volumes:

      - firefly_iii_upload:/var/www/html/storage/upload

    env_file: .env

    ports:

      - 5697:5697

    depends_on:

      - db

  db:

    image: mariadb

    hostname: fireflyiiidb

    restart: always

    environment:

      - MYSQL_RANDOM_ROOT_PASSWORD=yes

      - MYSQL_USER=firefly

      - MYSQL_PASSWORD=pipiriukai

      - MYSQL_DATABASE=firefly

    volumes:

      - firefly_iii_db:/var/lib/mysql

volumes:

   firefly_iii_upload:

   firefly_iii_db:
```

**Comment by JC5** [comment, marked as answer]
You have to change the first one only, so `5697:8080`.

**Reply by mariusCZ** [reply]
Still the same error unfortunately.

**Reply by JC5** [reply]
I see now, `Operation not permitted`. This may be something your shared hoster can fix for you?

**Reply by mariusCZ** [reply]
Not sure, got in touch with them but I am not counting on it. I presume this means it can't be changed on Docker then? If not, could it be changed on a normal install?

**Reply by JC5** [reply]
I'm not sure. This is a limitation on your hoster's side.

**Reply by mariusCZ** [reply]
Does this mean that the default port is not set by your app but rather by the system's apache configuration?

**Reply by JC5** [reply]
Exactly. The docker config is changed as was described earlier, outside of Docker Firefly III does not set the port.

**Reply by mariusCZ** [reply]
I see, thank you very much for your help!
